packages/containers/psammead-timestamp-container/src/index.test.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { isNull, shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import { latin } from '@bbc/gel-foundations/scripts';
+import { jalaali } from '@bbc/psammead-calendars';
```

Comment:
The tests here do not really need to know anything about the Jalaali calendar. Rather than use the real implementation, for greater test isolation I would create a simple `mockCalendar` object here that provides the `formatDate` function with a predictable result.